abd-clam/src/core/cluster/_cluster.rs

```diff
@@ -894,7 +884,7 @@ mod tests {
         let mut c1 = Cluster::new_root(&data, Some(42));
         c1.history = vec![true, true, false, false, true];
 
-        let s1 = SerializedCluster::from_cluster(&c1);
+        let (s1, s1_children) = SerializedCluster::from_cluster(&c1);
         let s1_string = serde_json::to_string(&s1).unwrap();
 
         let s2: SerializedCluster = serde_json::from_str(&s1_string).unwrap();
```

Comment:
Rename `s1` and `s2` to indicate which came from the in-memory cluster and which was created from the json-string.